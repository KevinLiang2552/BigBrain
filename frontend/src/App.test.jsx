// import React from 'react';
// import { render, screen } from '@testing-library/react';
// import App from './App';

// test('renders learn react link', () => {
//   render(<App />);
//   const linkElement = screen.getByText(/learn react/i);
//   expect(linkElement).toBeInTheDocument();
// });

import testErrorModal from './tests/testErrorModal.js';
import testPasswordInput from './tests/testPasswordInput';
import testActiveQuizItem from './tests/testActiveQuizItem';
import testQuizTime from './tests/testQuizTimer';
import testTopPlayerTable from './tests/testTopPlayerTable';

testErrorModal();
testPasswordInput();
testActiveQuizItem();
testQuizTime();
testTopPlayerTable();
