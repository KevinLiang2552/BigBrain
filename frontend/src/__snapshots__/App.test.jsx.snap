// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Modal Error modal renders correctly 1`] = `ShallowWrapper {}`;

exports[`active quiz item Active quiz item renders correctly 1`] = `ShallowWrapper {}`;

exports[`password input Password Input renders correctly 1`] = `ShallowWrapper {}`;
